spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageLocation.java

```diff
@@ -188,9 +186,9 @@ public String toString() {
 
 	private static String getBlobPathFromUri(URI gcsUri) {
```

Comment:
@dzou Thinking about this a bit more, why wouldn't one just do `gcsUri.toString()`?